modules/ossm-running-v2-v3-multitenant-deployment-model.adoc

```diff
@@ -0,0 +1,118 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/install/ossm-v2-v3-side-by-side-assembly.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-running-v2-v3-multitenant-deployment-model_{context}"]
+= Running {SMProductName} 2.6 and {SMProductName} 3 using multi-tenant deployment model
+//TP1 content influx. Title, etc may change.
+//No IA as of 10/23/2024 so this content is likely to move.
+
+If you are moving from {SMProductName} 2.6 from the default multi-tenant deployment model, you can run {SMProduct} 2.6 side-by-side with {SMProduct} 3.0, in one cluster, without them interfering with each other.
+
+In {SMProduct} 2.6, you can check your deployment model from the `ServiceMeshControlPlane` under `spec.mode`:
+
+.Example `ServiceMeshControlPlane` yaml
+[source, yaml]
+----
+apiVersion: maistra.io/v2
+kind: ServiceMeshControlPlane
+metadata:
+  name: basic
+spec:
+  mode: MultiTenant
+----
+
+//NOTES: 10/23/2024
+
+//DO NOT USE VERSION NUMBER ATTRIBUTE. It is set for 3.0, will likely change, and it is unclear if later versions would impact this procedure for moving from 2.6.
+//Only moving from 2.6 has been tested and verified by QE.
+//Mulitenant has no changes to OSSM 2.6, separate module.
+
+.Prerequisites
+
+* You are running {ocp-product-title} 4.14 or later.
+* You are running {SMProductName} 2.6.
+
+If you are not running {SMProduct} 2.6, you must upgrade to 2.6 before following this procedure. To upgrade to {SMProduct} version to 2.6, see: link:https://docs.openshift.com/container-platform/4.17/service_mesh/v2x/upgrading-ossm.html[Upgrading Service Mesh 2.x]
+//likely will need to link: to OSSM 2.x upgrade content
```

Comment:
Consider making this a NOTE or IMPORTANT block and putting it up in the intro above the prereqs. If you don't want to move it, add a `+` so that it attaches to the Service Mesh 2.6 bullet :nerd_face: